lib/lists/helpers.js

```diff
@@ -0,0 +1,73 @@
+import _ from 'lodash';
+import logger from '../utils/log';
+
+const log = logger(__filename);
+
+/**
+ * @param {array} items
+ * @param {string | object} testItem
+ * @param {string} testProperty (optional)
+ * @returns {integer}
+ */
+export function getItemIndex(items, testItem, testProperty) {
+  let listItemKeys, testItemKeys, index, matchObjects;
+
+  const tester = _.head(items);
+
+  if (_.isString(tester) && _.isObject(testItem) ||
+      _.isObject(tester) && _.isString(testItem) && !testProperty) {
+    log.error('The item you are looking for does not have the same data structure as the items in the list.', {action: 'modifyList'});
+    return false;
+  }
+
+  if (_.isObject(tester) && _.isObject(testItem)) {
+    listItemKeys = _.keys(_.head(items)).sort();
```

Comment:
You don't need to do `_.head(items)` here again, because you're using `tester`, right?